recipes/docling/recipe.yaml

```diff
@@ -0,0 +1,80 @@
+context:
+  name: docling
+  version: 2.6.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/docling-${{ version }}.tar.gz
+  sha256: 282755472a1a1d4a4088d794c6a89f399a5b7fbcea8b5f45012c441670b538d5
+  patches:
+    # lower the scipy lower bounds to match with c-f and avoid conflicts
+    - fix-deps.patch
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - docling = docling.cli.main:app
+
+requirements:
+  host:
+    - python ${{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - pydantic >=2.0.0,<3.0.0
+    - docling-core >=2.3.0,<3.0.0
+    - docling-ibm-models >=2.0.3,<3.0.0
+    - deepsearch-glm >=0.26.1,<0.27.0
+    - filetype >=1.2.0,<2.0.0
+    - pypdfium2 >=4.30.0,<5.0.0
+    - pydantic-settings >=2.3.0,<3.0.0
+    - huggingface_hub >=0.23,<1
+    - requests >=2.32.3,<3.0.0
+    - easyocr >=1.7.0,<2.0.0
+    - docling-parse >=2.0.2,<3.0.0
+    - certifi >=2024.7.4
+    - rtree >=1.3.0,<2.0.0
+    - scipy >=1.13.1,<2.0.0
+    - pyarrow >=16.1.0,<17.0.0
+    - typer >=0.12.5,<0.13.0
+    - python-docx >=1.1.2,<2.0.0
+    - python-pptx >=1.0.2,<2.0.0
+    - beautifulsoup4 >=4.12.3,<5.0.0
+    - pandas >=2.1.4,<3.0.0
+    - marko >=2.1.2,<3.0.0
+    - openpyxl >=3.1.5,<4.0.0
```

Comment:
I still think this is pretty impossible to keep in sync with upstream changes unless it's alphabetized 🤷 
(but as long as you want to deal with this extra overhead, be my guest).

Reply:
I am following the original repo order. In my experience, it makes it easier to sync the deps.